server/server.js

```diff
@@ -75,6 +75,19 @@ const appReady = app.prepare().then(() => {
     next()
   })
 
+  // robots.txt
+  if (process.env.NODE_ENV === 'production') {
+    server.get('/robots.txt', function (req, res) {
+      res.type('text/plain')
+      res.send('User-agent: *\nDisallow: /\nAllow:/home')
```

Comment:
You will want to allow most non signed in pages

Reply:
deferring to use allow as its not a standard and only googlebot knows about it.